packages/cli/index.js

```diff
@@ -23,4 +23,5 @@ yargs
   .command(setCmdName('init', init))
   .command(setCmdName('release', rollout))
   .command(setCmdName('intl', intl))
+  .command(setCmdName('format', pedantic))
```

Comment:
lol

although i think "lint" might be a better name here if the default behavior is to check instead of write. or `--fix` instead of `--write` to match eslint or something

Reply:
I went with just matching the underlying tools api, which both are what prettier's API is. Agree lint is a bit more descriptive but i think maybe confusing since this doesn't also do eslint tho maybe it should?

Reply:
well, we can always cut a breaking change later